python/main.py

```diff
@@ -1,8 +1,11 @@
 import inspect
 import json
 
-from sidecar import chat, cli, ingest, rewrite, storage, search
-from sidecar.typing import CliCommands
+from sidecar import chat, cli, ingest, rewrite, search, storage
+from sidecar._typing import CliCommands
+from sidecar.http import app
+
+app
```

Comment:
what's going on here? is this a no-op?

Reply:
Effectively. Really it was just me being lazy and wanting to simply run `uvicorn main:app` to run the server. After a couple tries of unsuccessfully getting it to run from within `http.py` I just decided to import it here for the sake of a prototype.